test/helpers/filesystem.py

```diff
@@ -26,7 +25,4 @@ def mount_tmpfs(path: Path, *, megabytes: int) -> Iterator[Path]:
 
         yield sub_dir
     finally:
-        # Delete all files in the tmpfs filesystem before unmounting it.
-        with suppress(Exception):
-            LocalPath(sub_dir).remove(rec=1)
-            subprocess.check_call(["sudo", "umount", str(sub_dir)])
```

Comment:
ended up removing this, because it was raising an exception and pytest was  complaining since it was not able to delete temporary directories